binding-hs/binding-hs.cabal

```diff
@@ -0,0 +1,51 @@
+name:                binding-hs
```

Comment:
check out hpack (which comes with stack) to generate a package.yaml. These are cleaner and easier to maintain.